interfaces/builtin/u2f_devices.go

```diff
@@ -116,6 +116,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "1d50",
 		ProductIDPattern: "60fc",
 	},
+	{
+		Name:             "MIRKey",
+		VendorIDPattern:  "0483",
+		ProductIDPattern: "a2ac",
```

Comment:
could you provide some documentation that the MIRKey has these vid and pid? I couldn't find anything with a quick search but perhaps I just didn't look hard enough

Reply:
I work for ellipticSecure who is the manufacturer of the MIRKey. ST (the VID owner) has sub licensed the PID for the device in 2018.
Do you want me to forward the official documentation to you, or did you just want me to confirm this?

Reply:
I just don't see this particular vid/pid documented in i.e. http://www.linux-usb.org/usb.ids or somewhere else that could be referenced, ideally some way that could be put publicly in this PR or in the code itself for others to independently verify that it is correct

Reply:
Please see https://github.com/Yubico/libfido2/blob/master/udev/fidodevs
Thanks

Reply:
thanks!

Reply:
Ok, I see the CLA check failed again, this time less catastrophically, but still.
I signed the CLA though - used my GitHub user name and email address in the form (email address is the same as on Launchpad). 
Anything I missed or should I just try it again?
Btw. - the CLA check leaks email addressed in the log.

Reply:
well the CLA check logs are only accessible to folks who are signed into github, and it's just the email that was used in committing that is freely visible to anyone who clones the repository so IMHO it's not a big deal.

Anyways yes I see the CLA check failed, I will retry it in a few hours in case things are a bit slow on propagating through LP before it works.

Reply:
@kobusgrobler Thanks for this PR, looks really great. I will chase the CLA internally here to see what is going on :)

Reply:
I received an email from Canonical that states I'm now a CLA member, so I guess the cla-check should pass now.

Reply:
Indeed, check is happy now, thank you for your patience :)